### PR TITLE
PSP-4135:Activity Status Changes (Matrix)

### PR DESCRIPTION
**Pull request by Maheshbabu28**

### CODE REVIEW

File: frontend/src/features/properties/map/activity/detail/ActivityContainer.tsx [hunk 1]
```diff
@@ -72,6 +72,7 @@ export const ActivityContainer: React.FunctionComponent<IActivityContainerProps>
 
   const onSaveActivityProperties = async () => {
     if (file?.fileType && !!activityModel) {
+      console.log('change', activityModel.toApi());
```
Comment (FuriousLlama) [inline frontend/src/features/properties/map/activity/detail/ActivityContainer.tsx:75]: log

Comment (Maheshbabu28) [reply frontend/src/features/properties/map/activity/detail/ActivityContainer.tsx:75]: fixed

File: frontend/src/features/properties/map/activity/detail/ActivityForm.tsx [hunk 2]
```diff
@@ -49,6 +49,7 @@ export const ActivityForm: React.FunctionComponent<IActivityFormProps> = ({
     <Formik<Api_Activity>
       initialValues={activity ?? { description: '', activityDataJson: '' }}
       onSubmit={async (values, formikHelpers) => {
+        values.activityStatusTypeCode = { id: values.status };
```
Comment (asanchezr) [inline frontend/src/features/properties/map/activity/detail/ActivityForm.tsx:52]: we have a helper to assist with this - **toTypeCode** or something like that

File: frontend/src/features/properties/map/activity/detail/ActivityControlsBar.tsx [hunk 3]
```diff
@@ -17,16 +21,26 @@ export const ActivityControlsBar: React.FunctionComponent<IActivityControlsBarPr
...
       {hasClaim(Claims.ACTIVITY_EDIT) && (
         <RightAlignedRow>
-          {!editMode && <EditButton onClick={() => setEditMode(true)} />}
+          <SectionField label="Status">
+            <Select
+              disabled={!editMode && !hasClaim(Claims.PROPERTY_EDIT)}
+              field="status"
+              options={activityStatusTypeOptions}
+            />
+          </SectionField>
```
Comment (asanchezr) [inline frontend/src/features/properties/map/activity/detail/ActivityControlsBar.tsx:36]: question: I understand that you want to disable the `<select>` when in view mode but why check for claim PROPERTY_EDIT as well? Shouldn't the activity status be driven by claim ACTIVITY_EDIT instead? Not sure if the story calls for this

**Comment by devinleighsmith** [comment]
During the demo I noticed that the status dropdown had centered the "Related Properties" button, which does not match the mockup.

File: frontend/src/features/properties/map/activity/detail/ActivityForm.tsx (hunk 2 above)
Comment (Maheshbabu28) [reply frontend/src/features/properties/map/activity/detail/ActivityForm.tsx:52]: that's helpful, done

File: frontend/src/features/properties/map/activity/detail/ActivityControlsBar.tsx (hunk 3 above)
Comment (Maheshbabu28) [reply frontend/src/features/properties/map/activity/detail/ActivityControlsBar.tsx]: Rremoved

**Pull request merged**